### DISCUSSION TITLE
My CrowdSecurity container keeps erroring out

### DISCUSSION THREAD

**Discussion by privatesam** [ZoeyVid/NPMplus#985] [Q&A] [answered]
This is the error message:
`time="2024-07-30T17:08:54+01:00" level=fatal msg="crowdsec init: while loading acquisition config: while configuring datasource of type appsec from /etc/crowdsec/acquis.d/npmplus.yaml (position: 3): failed to configure datasource appsec: unable to load appsec_config : no appsec-config found for crowdsecurity/virtual-patching"`

**Comment by Zoey2936** [comment]
should be fixed now in the readme

**Comment by privatesam** [comment]
Thanks for the quick response but now getting: 
`time="2024-07-30T17:34:28+01:00" level=fatal msg="crowdsec init: while loading acquisition config: while configuring datasource of type appsec from /etc/crowdsec/acquis.d/npmplus.yaml (position: 3): failed to configure datasource appsec: unable to load appsec_config : no appsec-config found for crowdsecurity/appsec-default"`

**Reply by Zoey2936** [reply]
sorry, https://github.com/crowdsecurity/hub/pull/1085 still needs to be merged

**Reply by Zoey2936** [reply]
nothing i can do

**Reply by privatesam** [reply]
OK no problem.  Thank you so much for your help and for trying to solve so quickly!

**Reply by Zoey2936** [reply, marked as answer]
Should work now

**Comment by privatesam** [comment]
Perhaps the crowdsec container has changed its file structure? 
![Screenshot 2024-07-30 173755](https://github.com/user-attachments/assets/04d9a24d-b01f-464f-a21d-373132741fdc)
